### PR TITLE
Removed gradients during evaluation

### PR DESCRIPTION
**Pull request by JacobB33**
The decode_and_plot function called during evaluation had gradients attached to the output image, so I added torch.no_grad() to remove them

### CODE REVIEW

**Comment by zchuning** [comment]
Good catch!
